crates/nix_rs/src/command.rs

```diff
@@ -119,6 +124,13 @@ impl NixCmd {
         cmd
     }
 
+    /// Suppress logs related to `override-input` usage by reducing `nix` command's verbosity
```

Comment:
Is this (_Suppress logs related to `override-input` usage_) the **only** effect of using `--quiet --quiet` or does it suppress anything else?

Reply:
If it does more than that, then function (`mute_override_input_logs`) wouldn't be accurate.

Reply:
Logically, it reduces verbosity level of the logs. I could rename the function to depict that.